content/docs/04.5-devx/01-dev-land-explore.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Land and Expand"
+metaTitle: "Palette Dev Engine for Free User"
+metaDescription: "Explore Palette Dev Engine as Free User"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import InfoBox from 'shared/components/InfoBox';
+import WarningBox from 'shared/components/WarningBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# Overview
+
+Palette is offering complimentary compute resources to the developers who sign up for Palette Dev Engine. These developers can use this compute resources to either, 
+
+* Provision a nested cluster to get full access to all the namespaces
+
+
+* Deploy an application (profile) which would in turn spin up a nested cluster
+
+In either case, they are governed by the quota set by Spectro Cloud. There is no limit to the number of application profiles they can model. 
```

Comment:
no need to mention for app modeling again. Also our quota we can speak up here or link to the quota what we are providing

Reply:
Let's make it clear what free is about : “We are offering 12 vCPU, 16 GiB of memory, and 20 GiB of storage for free”